checks/security_policy.go

```diff
@@ -17,16 +17,18 @@ package checks
 import (
 	"strings"
 
-	"github.com/ossf/scorecard/checker"
+	"github.com/ossf/scorecard/lib"
 )
 
+const securityPolicyStr = "Security-Policy"
```

Comment:
In `go` `securityPolicyStr` suffix aren't recommended in the names.

Reply:
Without suffix, there is less clarity. I think this confusion comes from check names like SecurityPolicy. So, i think keeping this as-is is fine for clarity, also we should think of renaming check function from like SecurityPolicy -> SecurityPolicyCheck to be more clear [https://github.com/ossf/scorecard/blob/main/checks/security_policy.go#L27]

Reply:
Ack. Leaving the suffix as is for now.

Reply:
How about renaming this to "checkName" everywhere? That communicates the purpose of this variable a lot better IMO, rather than just a variable name which mirrors the actual value.